ruddock/modules/rotation/helpers.py

```diff
@@ -0,0 +1,60 @@
+import flask
+import sqlalchemy
+
+def get_dinners():
+  query = sqlalchemy.text("""
+    SELECT DISTINCT dinner FROM rotation_prefrosh
+  """)
+  return flask.g.db.execute(query).fetchall()
+
+def get_prefrosh(prefrosh_id):
+  query = sqlalchemy.text("""
+    SELECT prefrosh_id, first_name, preferred_name, last_name, dinner,
+    bucket, votes_neg_two, votes_neg_one, votes_zero, votes_plus_one,
+    votes_plus_two, votes_plus_three, comments
+    FROM rotation_prefrosh WHERE prefrosh_id = {}
+    """.format(prefrosh_id))
```

Comment:
Don't do this. Formatting the SQL query directly is the easiest way to introduce sql injection vulnerabilities. Imagine what happens in this case:
```python
# Example of malicious user input
prefrosh_id = "1; DELETE FROM members;"
query = sqlalchemy.text("SELECT * FROM rotation_prefrosh WHERE prefrosh_id = {}".format(prefrosh_id))
flask.g.db.execute(query)
```
Instead, you should write queries like this:
```python
query = sqlalchemy.text("SELECT foo FROM foo_table WHERE bar = :b")
flask.g.db.execute(query, b=bar_value)
```
The `sqlalchemy.text` preprocessor compiles the query into a parameterized query, such that the database knows that the `:b` placeholder for some input and will not modify the structure of the sql query.

I'd highly suggest reading this for more info on how these things work:
https://en.wikipedia.org/wiki/SQL_injection